### DISCUSSION TITLE
AIR - Windows 11 LTSC support ?

### DISCUSSION THREAD

**Discussion by Andy-S27** [airsdk/Adobe-Runtime-Support#3622] [Q&A] [answered]
Apologies in advance if this is an irrelevant question.  It's my first post, I'm not technical and don't personally code using AIR but am a Configuration Mgr for an organisation running a multitude of software and am reaching out to all vendors to ask the same question regarding their products.  We are planning to move from Windows 10 Enterprise to Windows 11 Enterprise LTSC edition  ahead of Microsoft's cessation of support for Windows 10 in Oct 2025.  The question therefore is whether AIR runtime will function and is supported in Windows 11 LTSC.  We're currently running v33.1.1.  I can't find any reference to LTSC in the release notes nor any reference to LTSC elsewhere on the forums.

**Comment by ajwfrost** [comment, marked as answer]
Hi

There's no reason that AIR wouldn't run on this.. the only issues we've found with people using Windows 11 is the "N" edition which lacks the media support files (which can be downloaded anyway, see [this info](https://support.microsoft.com/en-us/windows/media-feature-pack-for-windows-10-11-n-september-2022-78cfeea5-c7d9-4aa8-b38f-ee4df1392009) from Microsoft).

FYI the shared AIR runtime is currently provided as a 32-bit binary (so goes into "Program Files (x86)") - at some point we will switch it over to 64-bit, but we do still have some people running older 32-bit versions of Windows. For people developing AIR applications that include the "captive" runtime (i.e. an AIR-based application that is self-contained and doesn't require the shared AIR runtime), it's been possible to create this as 32-bit or 64-bit for quite some time.

Hope that helps! I'm 99% sure you'll be fine but if you do run into any issues, please raise them here.

thanks

**Reply by Andy-S27** [reply]
Hi Andrew, great and thanks for that reassuring detail and responding so quickly too.  It certainly does sound like we'll be OK but will bear this post in mind if we do experience any future issues and will be back in touch.  Kind regards, Andy
